components/automate-ui/src/app/components/notification/notification.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component, Input, HostBinding, Output, EventEmitter, AfterViewInit } from '@angular/core';
+import { trigger, transition, style, animate } from '@angular/animations';
+import { Type } from 'app/entities/notifications/notification.model';
+
+const slideDownAnim = trigger('slideDownAnim', [
+  transition(':enter', [
+    style({
+      opacity: 0,
+      transform: 'translateY(-100%) scale(0.98)'
+    }),
+    animate('300ms ease-out',
+      style({
+        opacity: 1,
+        transform: 'translateY(0%) scale(1)'
+      }))
+  ]),
+  transition(':leave', [
+    style({ opacity: 1 }),
+    animate('195ms ease-in', style({ opacity: 0 }))
+  ])
+]);
```

Comment:
This piece is controlling the animations of the notification ribbons.  Each of these properties are cheap, using translate and scale, neither of which cause a reflow.